rest/views.py

```diff
@@ -48,6 +49,39 @@
 from django.http.response import Http404
 
 
+def get_urls(raw_urls, list_urls, urlbase=''):
+    """
+    Recursively builds a list of all the urls in the current project and
+    the name of their associated view
+    """
+    list_urls = list_urls or []
+    from operator import itemgetter
```

Comment:
yeah, putting it on top of file gave me error on travis, but was working fine on local env.